### PR TITLE
update token location

### PR DESCRIPTION
**Pull request by amasad**
@basicer

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-const readline = require("readline");
-const WebSocket = require("ws");
-const chokidar = require("chokidar");
-const fs = require("fs");
-const qs = require("querystring");
+const request = require('request-promise');
+const payloads = require('./payloads.js');
+const readline = require('readline');
+const WebSocket = require('ws');
+const chokidar = require('chokidar');
```
Comment (amasad) [inline client.js:5]: it's all prettier

File: client.js [hunk 2]
```diff
@@ -17,21 +19,31 @@ class Client {
...
   async auth() {
     let resp = await request({
-      url: "https://repl.it/languages/" + this.payload.lang,
+      url: `${domain}/languages/${this.payload.lang}`,
       headers: {
         //'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36'
-        "User-Agent": "Mozilla/5.0 (repl.sh)"
+        'User-Agent': 'Mozilla/5.0 (repl.sh)',
       },
-      simple: false,
```
Comment (amasad) [inline client.js:25]: why did you add this? Isn't it better if it errors?

**Review by basicer — APPROVED** [review]
👍
